vnext/Microsoft.ReactNative.Cxx/ReactCommon/TurboModuleUtils.cpp

```diff
@@ -0,0 +1,98 @@
+// This file is a copy from react-native NPM package to avoid dependency on folly.
```

Comment:
how do we keep this in sync with what's in the npm package? we should try to avoid forking as much as we can

Reply:
Agree. Per @NickGerleman  suggestion I have created a PR in react-native repo to avoid forking: https://github.com/facebook/react-native/pull/30672.